pkg/controller/volume/scheduling/scheduler_binder.go

```diff
@@ -469,9 +469,14 @@ func (b *volumeBinder) checkBindings(pod *v1.Pod, bindings []*bindingInfo, claim
 		return false, fmt.Errorf("failed to get cached claims to provision for pod %q", podName)
 	}
 
-	node, err := b.nodeInformer.Lister().Get(pod.Spec.NodeName)
-	if err != nil {
-		return false, fmt.Errorf("failed to get node %q: %v", pod.Spec.NodeName, err)
+	var node *v1.Node
+	var err error
+	for _, nodeInformer := range b.nodeInformers {
+		node, err = nodeInformer.Lister().Get(pod.Spec.NodeName)
+		if err != nil { // TODO - check error type, continue to search next RP if no found
```

Comment:
likely the TODO is needed in this PR instead of later. e.g. if the node is on the second RP, this function currently will return error.

Reply:
This is in volumeBinder checkBindings. Currently I don't have a use case to test what will be the error message so don't have a way to handle. I will add an error log for now.